capstone/src/main/java/com/google/univiz/UniversityData.java

```diff
@@ -0,0 +1,67 @@
+package com.google.univiz;
+
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+public abstract class UniversityData {
+
+  public static UniversityData create(
+      String name,
+      String city,
+      float latitude,
+      float longitude,
+      int flagMainCampus,
+      int urbanizationDegree,
```

Comment:
what does urbanizationDegree mean? would definitely be worth adding javadoc about this...